src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,118 @@
+.page__cards {
+  margin-top: 50px;
+}
+
+.cards {
+  @include flex();
+  flex-wrap: wrap;
+
+  &__container {
+    width: cardWidth($cardsWidth, $cardsCardMargin);
+    @include flex();
+    flex-wrap: wrap;
+    
+  }
+
+  &__card {
+    margin-left: $cardsCardMargin;
+    margin-right: $cardsCardMargin;
+    margin-bottom: 50px;
+  }
```

Comment:
let's leave only margin-bottom
then use justify-content property on flex container to add space between cards